config/install/field.storage.node.field_species.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_species
+field_name: field_species
+entity_type: node
+type: entity_reference
+settings:
+  target_type: node_type
```

Comment:
This needs to reference "Content" instead of "Content Type" in the field settings.
<img width="353" alt="people_species_field_storage" src="https://github.com/notS0Solo/swapimigration/assets/1434979/164c14aa-ab5a-4565-ac9c-3ffc30315314">
Once that change is made, update the field configuration to reference the "Species" content type as well.

Reply:
Done and ready for review.